media/css/mozorg/home/home-b.scss

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/buttons';
+
+body {
+    @include open-sans;
+    border-top: 0;
+}
+
+h2,
+h3 {
+    @include zilla-slab;
+}
+
+// hide download button in the nav
+#nav-download-firefox,
+#global-nav-download-firefox {
+    display: none;
```

Comment:
I'm guessing this is intentional as per the design, but fwiw If I'm signed into Sync on Nightly, then it's a little hard to download Firefox directly from the home page. Not a huge issue for most users, but I we've had bug reports like this in the past when conditional states on different pages cause frustration.

Reply:
Sigh, yeah, the million edge cases are why we all loathe conditional content, particularly around download buttons. Perhaps I could check for release channel as well?

Reply:
Totally up to you and how complicated it gets, maybe something we could fix later?

Reply:
Yep, after looking at it, going to leave as-is. Will comment in the JS file that it should be re-visited if/when the logic is re-addressed.